### DISCUSSION TITLE
Clublog integration

### DISCUSSION THREAD

**Discussion by mboroff** [wavelog/wavelog#1037] [Q&A] [answered]
I have just setup wavelog on a Pi 5. I am able to manually log and import my adit file from WSJT-X. I am at the point I want to add uploads to club log in the crontab. I am following these directions:
:Next, you can test it works by going to <url-to-wavelog> /index.php/clublog/upload/> you should see a success message if it worked, Wavelog will need write access to the /uploads folder."

There is no clublog folder below my folder with index.php. that is /var/www/html therefore the file is not found.

What am I missing on my configuration to create the clublog folder. I have don the administration to add my id and password on clublog.

**Comment by HB9HIL** [comment, marked as answer]
Hello @mboroff 

the hint with "..by going to `/index.php/clublog/upload` does not describe a folder path. It describes the LINK you should visit

`https://<your wavelog url>/index.php/clublog/upload`

**Reply by mboroff** [reply]
I understand that it is a path. I believe it implies a folder clublog is in the path and it is not on my Pi. Did I miss something in my install process. 

My server is located at 10.0.0.127 so http://10.0.0.127/index.php/clublog/upload can’t find the server. HTTP results in file not found.

it is my impression that Apache sets the too to /var/www/html which is where index.php is located and how I bring up wavelog. Should I not see a file structure of /var/www/html/clublog?

**Reply by mboroff** [reply]
My server is at 10.0.0.127. Using your address to my server would be https://10.0.0.127/index.php/clublog/upload yields failed to open page.

When I login to wave log my url is http://10.0.0.127/html/index.php/dashboard .

**Reply by HB9HIL** [reply]
The the correct URL would be [http://10.0.0.127/html/index.php/clublog/upload](http://10.0.0.127/html/index.php/clublog/upload) 😉

**Reply by mboroff** [reply]
I ran that from my phone and I got a message “Clublog upload for WD9GYM”.  So why did my typing that into my browser work? It looks the same to me.

**Reply by mboroff** [reply]
NeverMind. I see the missing html in my link.
